noetic/ros-blender/Dockerfile

```diff
@@ -0,0 +1,99 @@
+#
+# Docker file for base ROS+Blender install
+# No ROS nodes are started.
+#
+# The primary difficulty here is that Blender wants Python3
+# while ROS wants Python2, and so there are install conflicts.
+#
+# To build:
+# docker build -t opencog/ros-noetic-blender .
+#
+# To run:
+# docker run --rm --privileged -i -v /tmp/.X11-unix/X0:/tmp/.X11-unix/X0 \
+#    -v /dev/dri:/dev/dri -v /dev/shm:/dev/shm -e DISPLAY=:0.0 \
+#    -t opencog/ros-noetic-blender
+#
+# The above should be sufficient to get blender running with full
+# 3D hardware acceleration on the local host (dri==direct rendering interface)
+# Note: /dev/dri=gpu, /dev/shm=gpu
+#
+FROM opencog/ros-noetic-base
+MAINTAINER Mandeep Singh Bhatia "mandeep.singh.bhatia@gmail.com"
+MAINTAINER David Hart "dhart@opencog.org"
+MAINTAINER Linas Vepštas "linasvepstas@gmail.com"
```

Comment:
Again, you should add your name as the latest and newest maintainer :-)  Appearantly, there can be more than one!